beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md

```diff
@@ -10,16 +10,16 @@ To use this view, complete the following steps:
 
 1. [Setup Project Griffon](../set-up-project-griffon.md)
 2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Project Griffon session
-3. In the **Home** view, select **Custom Validation**
+3. In the **Home** view, select **Validation Scripts** (to be deprecated for **Validation Summary**)
```

Comment:
Can we remove the parenthesis here? If this will change in the future we can always come back to update the docs